plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -45,20 +45,50 @@ import {
 } from '@backstage/core-components';
 
 export const SPLUNK_ON_CALL_TEAM = 'splunk.com/on-call-team';
+export const SPLUNK_ON_CALL_ROUTING_KEY = 'splunk.com/on-call-routing-key';
 
-export const MissingTeamAnnotation = () => (
-  <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_TEAM} />
+export const MissingAnnotation = () => (
+  <div>
+    <Typography>
+      The Splunk On Call plugin requires setting either the{' '}
+      <code>{SPLUNK_ON_CALL_TEAM}</code> or the{' '}
+      <code>{SPLUNK_ON_CALL_ROUTING_KEY}</code> annotation.
+    </Typography>
+    <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_TEAM} />
+    <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_ROUTING_KEY} />
+  </div>
 );
 
-export const InvalidTeamAnnotation = ({ teamName }: { teamName: string }) => (
-  <CardContent>
-    <EmptyState
-      title={`Could not find team named "${teamName}" in the Splunk On-Call API`}
-      missing="info"
-      description={`Escalation Policy and incident information unavailable. Please verify that the team you added "${teamName}" is valid if you want to enable Splunk On-Call.`}
-    />
-  </CardContent>
-);
+export const InvalidAnnotation = ({
+  teamName,
+  routingKey,
+}: {
+  teamName: string | undefined;
+  routingKey: string | undefined;
+}) => {
+  let titleSuffix = 'provided annotation';
+
+  if (routingKey) {
+    titleSuffix = `"${routingKey}" routing key`;
+  }
+
+  if (teamName) {
+    titleSuffix = `"${teamName}" team name`;
+  }
+
+  return (
+    <Card>
+      <CardHeader title="Splunk On-Call" />
+      <CardContent>
+        <EmptyState
+          title={`Splunk On-Call API returned no record of teams associated with the ${titleSuffix}`}
+          missing="info"
+          description="Escalation Policy and incident information unavailable. Splunk On-Call requires a valid team name or routing key."
+        />
+      </CardContent>
+    </Card>
```

Comment:
☝️ Note that the invalid annotation component now displays slightly different language, based on which annotation was used.